store/postgres/src/chain_store.rs

```diff
@@ -1690,12 +1698,15 @@ impl ChainStoreTrait for ChainStore {
             .confirm_block_hash(&conn, &self.chain, number, hash)
     }
 
-    fn block_number(&self, hash: &BlockHash) -> Result<Option<(String, BlockNumber)>, StoreError> {
+    fn block_number(
+        &self,
+        hash: &BlockHash,
+    ) -> Result<Option<(String, BlockNumber, Option<String>)>, StoreError> {
         let conn = self.get_conn()?;
```

Comment:
This would ideally be asyncified to use `with_conn`.

Reply:
this one is also used in a few places so I'd rather take that as a separate PR